<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>galeria</class>
 <widget class="QDialog" name="galeria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>800</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>33</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(191, 64, 64, 0);
background-repeat:none;
border:none;

</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Seleccionar imagen...</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="imagen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>901</width>
     <height>641</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="text">
    <string>                                                      Seleccione una imagen para mostrar</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="min1">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>125</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="min2">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>230</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="min3">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>335</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="min1_boton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>125</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(191, 64, 64, 0);
background-repeat:none;
border:none;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="min2_boton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>230</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(191, 64, 64, 0);
background-repeat:none;
border:none;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="min3_boton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>335</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(191, 64, 64, 0);
background-repeat:none;
border:none;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>801</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/imagenes/fondo_Alarmas.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>700</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/btn_Cerrar.png</normaloff>:/imagenes/btn_Cerrar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="trash">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>700</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/imagenes/btn_Eliminar.png</normaloff>:/imagenes/btn_Eliminar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>560</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ultimas Capturas</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>comboBox</zorder>
  <zorder>imagen</zorder>
  <zorder>min1</zorder>
  <zorder>min2</zorder>
  <zorder>min3</zorder>
  <zorder>min1_boton</zorder>
  <zorder>min2_boton</zorder>
  <zorder>min3_boton</zorder>
  <zorder>toolButton</zorder>
  <zorder>trash</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
